### PR TITLE
backupccl: add a few tracing spans to online restore

### PR DESCRIPTION
**Pull request by stevendanna**
This adds tracing spans around KV and status server requests. While we expect these calls to be fast, sometimes the world surprises us.

Release note: None
Epic: None

### CODE REVIEW

File: pkg/ccl/backupccl/restore_online.go [hunk 1]
```diff
@@ -271,28 +301,24 @@ func (r *restoreResumer) maybeCalculateTotalDownloadSpans(
 		return total, nil
 	}
 
+	ctx, sp := tracing.ChildSpan(ctx, "backupcc..maybeCalculateDownloadSpans")
```
Comment (msbutler) [inline pkg/ccl/backupccl/restore_online.go:304]: nit: sub the extra period . for a l

**Review by msbutler — APPROVED** [review]
Thanks for doing this!

**Comment by stevendanna** [comment]
bors r=msbutler

**Comment by stevendanna** [comment]
bors retry

**Pull request merged**